src/useListToken.js

```diff
@@ -4,15 +4,24 @@ import getToken from './lib/token';
 const TOKEN_NAME = 'token';
 
 export function getCurrentToken() {
-  return window.localStorage.getItem(TOKEN_NAME) || getToken();
+  let token = window.localStorage.getItem(TOKEN_NAME);
+
+  if (token) {
+    return token;
+  }
+
+  token = getToken();
+  window.localStorage.setItem(TOKEN_NAME, token);
```

Comment:
Wondering what necessitated this change? I wouldn't expect a function named `getCurrentToken` to _set_ the token.

Reply:
I don't recall changing this. Maybe it was changed when Nikema was working on it earlier this week?